package-lock.json

```diff
@@ -23,10 +23,11 @@
         "@ethersproject/strings": "^5.0.5",
         "@ethersproject/wallet": "^5.0.7",
         "@imtbl/imlogging": "^1.0.33",
-        "@imtbl/imx-sdk": "1.3.3",
+        "@imtbl/imx-sdk": "1.3.39",
         "@typescript-eslint/eslint-plugin": "^4.29.2",
         "@typescript-eslint/parser": "^4.28.0",
         "axios": "^0.21.2",
+        "axios-retry": "^3.2.4",
```

Comment:
Why is this one added?

Reply:
ahh, was an artefact for when i was using `yalc` to develop locally. has now been removed